robot/basestation/templates/elements/navigation/Navigation.html

```diff
@@ -1,12 +1,12 @@
 <h5 class="container-header">Navigation Data</h5>
-<div style="display: block" id="rover-unchanging">
+<div class="navigation" style="display: block" id="rover-unchanging">
```

Comment:
The `display: block;` should belong to the `navigation` class

Reply:
Fixed